presentations/en/2020-GCDevEx-DG-Comm.md

```diff
@@ -59,70 +130,89 @@ Red tape reduction initiative:
 
 ---
 
 ## This is not about increasing outsourcing
 
+Note:
+
+- probably some flags are coming up for you about outsourcing...our feeling is that this is not about increasing outsourcing but rather about doing procurement differently
+
 ---
 
-## What sort of changes are we talking about?
+## What do we expect to achieve?
+
+- increased use of open source at ESDC
+- access to developers who wouldn’t normally bid on GC IT contracts
+- reduced workload burden on ESDC procurement officers
+- increased knowledge of modern software/tools by IITB employees
+- increased capabilities around agile including breaking work up into smaller chunks (NOT CONTRACT SPLITTING)
 
-- Procurement process for LDV simplified:
-  - Fewer approvals required
-  - Automation
-- Culture change: priority given to procurement and invoicing activities
-- Web tool for posting opportunities
+Note:
+
+- simplifying the procurement of open source will encourage the use of open source (bit of a chicken/egg situation)
+- Regarding access to developers, 95% of SMEs surveyed in 2017 do not see the GC as a buyer.
+  - Through a **simplified and expedited** process that **reduces the barriers** that these organizations (including startups) and freelancers face, and a **targeted communications strategy** (to diverse businesses such as those that are Black, and Indigenous led), this pilot is **expected to increase access to IT talent that the GC is currently missing out on**
+- reduced workload burden on procurement officers - we propose that they will help put guardrails in place but largely be uninvolved in the micro-procurement process
+- Strategic procurement of open source code can bring knowledge into IT teams that is not currently there. Suppliers will be contracted to solve problems and when they turn over the open source code, ESDC IT teams will be able to learn from suppliers in almost a train the trainer sort of way. This would enable IITB to build capacity in-house and embed modern tools and practices within IITB teams.
+- Further, contributions to GC open source projects, or open source projects being leveraged by the GC, has the **potential to be leveraged as a recruitment tool** for IITB. (CDS has experienced this with COVID Alert)
 
 ---
 
 ## Alignment
 
 - ESDC Mandate
-- DOSP
+- Digital Operations Strategic Plan
 - Digital Nations Charter commitment
 - Directive on Service and Digital
 - ESDC Investment Plan
 
 Note:
 
-- there is strong alignment to ESDC mandate of "works to improve the standard of living and quality of life for all Canadians. We do this by promoting a labour force that is highly skilled. We also promote an efficient and inclusive labour market."
-- DOSP – Procurement Modernization, Agility, Open Source​
-- Digital Nations Charter commitment (formerly D9) – Open Markets ​
+- as mentioned, lots of people see the benefits of having separate processes micro-procurement
```

Comment:
_"... having separate processes micro-procurement"_

Rather "having a separate process for micro-procurement" ?